server/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
```

Comment:
@ammartinwala52 Hello Ammar I added a .gitignore on the server as well. since this project can be installed on different platform, node_modules is something you don't want to have in your repository: it's not the same on Linux or Windows and also depends on other factors such as your processor family and OS version / services.

Plus it's easier to review without it :smile: 

You might need to run `npm install` in your server folder after pulling this change